app/controllers/users_controller.rb

```diff
@@ -1,4 +1,6 @@
 class UsersController < ApplicationController
+  layout 'users'
```

Comment:
Here you don't need to specify the layout. By convention Rails looks for first the specific layout for this controller, if it's not present then uses the application layout. Please check: http://guides.rubyonrails.org/layouts_and_rendering.html Section `2.2.13 Finding Layouts`